### ISSUE TITLE
The template doesn't work out of the box

### ISSUE THREAD

**Issue by ctwhome** [michalzaq12/electron-nuxt#619]
On MacOS(bigSur 11.2.1), when installing the template with `vue init michalzaq12/electron-nuxt` gives the following error after running `yarn dev`

```shell
Loading...
ERROR in ./src/renderer/.nuxt/client.js
Module build failed (from ./node_modules/@nuxt/webpack/node_modules/babel-loader/lib/index.js):
Error: /Users/ctw/Sites/github/nuxt-electron/src/renderer/.nuxt/client.js: Cannot find module '@babel/preset-env/lib/utils'
Require stack:
- /Users/ctw/Sites/github/nuxt-electron/node_modules/@nuxt/babel-preset-app/src/polyfills-plugin.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/@nuxt/babel-preset-app/src/index.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/core/lib/config/files/module-types.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/core/lib/config/files/configuration.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/core/lib/config/files/index.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/core/lib/index.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/@nuxt/webpack/node_modules/babel-loader/lib/index.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/loader-runner/lib/loadLoader.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/loader-runner/lib/LoaderRunner.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/webpack/lib/NormalModule.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/webpack/lib/NormalModuleFactory.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/webpack/lib/Compiler.js
- /Users/ctw/Sites/github/nuxt-electron/node_modules/webpack/lib/webpack.js
- /Users/ctw/Sites/github/nuxt-electron/.electron-nuxt/renderer/nuxt.config.js
- /Users/ctw/Sites/github/nuxt-electron/.electron-nuxt/renderer/nuxt-process.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:997:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at PluginPass.Program (/Users/ctw/Sites/github/nuxt-electron/node_modules/@nuxt/babel-preset-app/src/polyfills-plugin.js:15:34)
    at newFn (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/visitors.js:175:21)
    at NodePath._call (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/path/context.js:55:20)
    at NodePath.call (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/path/context.js:42:17)
    at NodePath.visit (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/path/context.js:92:31)
    at TraversalContext.visitQueue (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/context.js:116:16)
    at TraversalContext.visitSingle (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/context.js:85:19)
    at TraversalContext.visit (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/context.js:144:19)
    at Function.traverse.node (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/index.js:82:17)
    at traverse (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/traverse/lib/index.js:62:12)
    at transformFile (/Users/ctw/Sites/github/nuxt-electron/node_modules/@babel/core/lib/transformation/index.js:107:29)
    at transformFile.next (<anonymous>)
```

**Comment by clayperez** [comment]
Dangit! Any work-arounds?

**Comment by michalzaq12** [comment]
Fixed in 1.7.0
